requirements.txt

```diff
@@ -32,6 +32,7 @@ polars>=0.18.0, <0.19.2
 protobuf~=4.21.12
 psutil==5.9.8
 pyarrow==14.0.1
+pyairtable>=2.3.3
```

Comment:
pyairtable is already added to requirements.txt. maybe you can rebase master and resolve conflict